### ISSUE TITLE
As special use applicant, when I go click to apply to the special use application from the noncommerical or temp outfitter guide, i should be informed that i will need to log into login.gov

### ISSUE THREAD

**Issue by lauraGgit** [18F/fs-permit-platform#115]
## Notes
Currently no indication that you'll have to use login.gov to auth into before applying from the learn more pages.

https://forest-service-trees-staging.app.cloud.gov/applications/noncommercial-group-use/learn-more

https://forest-service-trees-staging.app.cloud.gov/applications/temp-outfitters/learn-more

## Acceptance Criteria
- [ ]

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by lauraGgit** [comment]
https://login.usajobs.gov/Access/Transition

**Comment by hannahkane** [comment]
@lauraGgit - is there guidance around this? Asking because the "Login or create account" message in the header doesn't specify that the user will use login.gov specifically. Do you know if that's required or advised? (Or maybe @hursey013 knows the answer to that?)

Also wondering if, after being prompted to log in from the Apply buttons, we can take the now-authenticated user directly to the relevant application, rather than taking them back to the Guide page. If that's possible, I might suggest @msadakFS file another issue to address that (but wouldn't prioritize it high in the backlog).

**Comment by msadakFS** [comment]
@lauraGgit @hannahkane I created a new issue #215

**Comment by lauraGgit** [comment]
@hannahkane @msadakFS i posted the usa jobs link as it should be outside of the button, but it should be next to it, as the usajobs ui does.

**Comment by hannahkane** [comment]
@lauraGgit - why is this in the icebox?

**Comment by lauraGgit** [comment]
this should be in the blocked column, we are just waiting on Login.gov to deploy to prod with us.
